src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruitDB = new ArrayList<>();
```

Comment:
Let's use a data structure that enforces uniqueness. This list can technically contain two entries for the same fruit if aprogrammer makes a mistake at some point. For this reason, HashMap is advisable, where fruit, probably just string of it's name in lowercase, is the key, and quantity - value